migtests/scripts/resumption.py

```diff
@@ -0,0 +1,260 @@
+                        help="Path to the YAML configuration file")
+    return parser.parse_args()
+
+def load_config(config_file):
+    """Load the configuration from the provided YAML file."""
+    if not os.path.exists(config_file):
+        raise FileNotFoundError(f"Config file not found: {config_file}")
+    with open(config_file, 'r') as file:
+        config = yaml.safe_load(file)        
+    return config
+
+def prepare_import_data_file_command(config):
+    """
+    Prepares the yb-voyager import data file command based on the given configuration.
+    """
+    file_table_map = config['file_table_map']
+    additional_flags = config.get('additional_flags', {})
+
+    args = [
+        'yb-voyager', 'import', 'data', 'file',
+        '--export-dir', os.getenv('EXPORT_DIR', ''),
+        '--target-db-host', os.getenv('TARGET_DB_HOST', ''),
+        '--target-db-port', os.getenv('TARGET_DB_PORT', ''),
+        '--target-db-user', os.getenv('TARGET_DB_USER', ''),
+        '--target-db-password', os.getenv('TARGET_DB_PASSWORD', ''),
+        '--target-db-schema', os.getenv('TARGET_DB_SCHEMA', ''),
+        '--target-db-name', os.getenv('TARGET_DB_NAME', ''),
+        '--disable-pb', 'true',
+        '--send-diagnostics', 'false',
+        '--data-dir', os.getenv('DATA_DIR', ''),
+        '--file-table-map', file_table_map
+    ]
+
+    if os.getenv('RUN_WITHOUT_ADAPTIVE_PARALLELISM') == 'true':
+        args.extend(['--enable-adaptive-parallelism', 'false'])
+
+    for flag, value in additional_flags.items():
+        args.append(flag)
+        args.append(value)
+
+    return args
+
+
+def prepare_import_data_command(config):
+    """
+    Prepares the yb-voyager import data command based on the given configuration.
+    """
+
+    additional_flags = config.get('additional_flags', {})
+
+    args = [
+        'yb-voyager', 'import', 'data',
+        '--export-dir', os.getenv('EXPORT_DIR', ''),
+        '--target-db-host', os.getenv('TARGET_DB_HOST', ''),
+        '--target-db-port', os.getenv('TARGET_DB_PORT', ''),
+        '--target-db-user', os.getenv('TARGET_DB_USER', ''),
+        '--target-db-password', os.getenv('TARGET_DB_PASSWORD', ''),
+        '--target-db-name', os.getenv('TARGET_DB_NAME', ''),
+        '--disable-pb', 'true',
+        '--send-diagnostics', 'false',
+    ]
+    
+    if os.getenv('SOURCE_DB_TYPE') != 'postgresql':
+        args.extend(['--target-db-schema', os.getenv('TARGET_DB_SCHEMA', '')])
+
+    if os.getenv('RUN_WITHOUT_ADAPTIVE_PARALLELISM') == 'true':
+        args.extend(['--enable-adaptive-parallelism', 'false'])
+
+    for flag, value in additional_flags.items():
+        args.append(flag)
+        args.append(value)
+
+    return args
+
+
+def run_and_resume_voyager(command, resumption):
+    """
+    Runs the yb-voyager command with support for resumption testing.
+    """
+    for attempt in range(1, resumption['max_restarts'] + 1):
```

Comment:
let's get/define all the configs in the beginning. It will make it easier to understand what all configuration options are involved. 
``` 
max_restarts = resumption['max_restarts']
min_interrupt_seconds = resumption['min_interrupt_seconds']
... 
```